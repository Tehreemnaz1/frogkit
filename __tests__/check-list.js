import { mount } from 'vue-test-utils'
import Vue from 'vue'
import CheckList from '@/components/check-list'

describe('Check List component', () => {

  it('should render component and match snapshot', () => {
    const wrapper = mount(CheckList, {propsData: {placeholder: 'some text goes here' }})
    expect(wrapper.html()).toMatchSnapshot()
  })
  it('should render component with bold text', () => {
    const wrapper = mount(CheckList, {propsData: {placeholder: 'some text goes here', bold: true }})
    expect(wrapper.html()).toMatchSnapshot()
  })

})
