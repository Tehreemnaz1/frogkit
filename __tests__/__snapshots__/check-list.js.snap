// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check List component should render component and match snapshot 1`] = `
<section class="fk-check-list fk-check-list--orange">
    <flex align="top">
        <icon icon="checkmark" color="orange" class="fk-check-list__icon"></icon>
        <flex column="">
            <p class="fk-check-list__placeholder"> some text goes here </p>
        </flex>
    </flex>
</section>
`;

exports[`Check List component should render component with bold text 1`] = `
<section class="fk-check-list fk-check-list--orange fk-check-list--bold">
    <flex align="top">
        <icon icon="checkmark" color="orange" class="fk-check-list__icon"></icon>
        <flex column=""><b class="fk-check-list__placeholder"> some text goes here </b> </flex>
    </flex>
</section>
`;
